### PR TITLE
SPARKC-414: Remove Dependency on CassandraUtil

### PR DESCRIPTION
**Pull request by RussellSpitzer**
We weren't using it anyway

### CODE REVIEW

File: spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/types/TimestampFormatter.scala [hunk 1]
```diff
@@ -2,7 +2,6 @@ package com.datastax.spark.connector.types
 
 import java.util.Date
 
-import org.apache.cassandra.serializers.TimestampSerializer
```
Comment (RussellSpitzer) [inline spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/types/TimestampFormatter.scala]: @JeremiahDJordan The full changes required to remove our dependency on cassandra-clientutil :\

Comment (JeremiahDJordan) [inline spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/types/TimestampFormatter.scala]: \o/

**Comment by jtgrabowski** [comment]
+1 
but we have (unrelated with this PR) problem with one test
```
com.datastax.spark.connector.writer.TableWriterSpec.A TableWriter should be able to append and prepend elements to a C* list
List("PrependTwo", "PrependOne", "One") did not contain all of ("PrependOne", "PrependTwo", "PrependThree") 
```
see failed jenkins logs for details

**Comment by RussellSpitzer** [comment]
retest this please

**Comment by RussellSpitzer** [comment]
@jtgrabowski It was just the wind I guess :)

**Comment by jtgrabowski** [comment]
;)

**Pull request merged**